prac_06/guitar.py

```diff
@@ -0,0 +1,23 @@
+
+CURRENT_YEAR = 2023
+VINTAGE_YEAR = 50
+
+
+class Guitar:
+    """Class for storing data and determining old guitar or not"""
+    def __init__(self, name="", year=0, cost=0):
```

Comment:
I think you should add a doc string comment under def__init__( )

Reply:
oki thank you !